test/foxStakingTest.ts

```diff
@@ -1,20 +1,25 @@
-import { ethers, deployments, getNamedAccounts } from "hardhat";
+import { ethers, deployments, getNamedAccounts, network } from "hardhat";
 import { expect } from "chai";
 import { Foxy } from "../typechain-types/Foxy";
 import { FoxStaking } from "../typechain-types/FoxStaking";
 import { StakingWarmup } from "../typechain-types/StakingWarmup";
-import { ERC20 } from "../typechain-types/ERC20";
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
-import { BigNumber, Signer } from "ethers";
+import { BigNumber, Contract, Signer } from "ethers";
+import { ERC20__factory, Foxy__factory } from "../typechain-types";
```

Comment:
something like `import ERC20 from "@openzeppelin/contracts/build/contracts/ERC20.json"; ` and `ERC20.abi` should work and be a little cleaner.

Reply:
Also not sure the `Foxy__factory` is being used.